google-map-directions.html

```diff
@@ -53,8 +58,10 @@
        * @type google.maps.Map
        * @default null
        */
-      map: null,
-
+      map: {
+        type: Object,
+        observer: 'mapChanged'
```

Comment:
Should we set some default values on these properties. The doc strings still say `@default null`